docs/posts/overview-of-ballerina-0.980.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Overview of Ballerina 0.980
+author: Kishanthan Thangarajah
+date: 26 July 2018
+status: Published
+abstract: Overview of new features and improvements added in ballerina 0.980.0 release
+socialmediaimage: ballet-sneaker-dress-ballet-dancer-163379.jpeg
```

Comment:
Is the intention to link to the image here? It does not seem linked to the image location.